.github/workflows/build.yml

```diff
@@ -35,7 +37,7 @@ jobs:
         id: get-product-version
         run: |
           echo "::set-output name=product-version::$(make version)"
-          echo "::set-output name=ldflags::-s -w -X github.com/hashicorp/terraform-aws-consul-lambda/version.GitCommit=$(git rev-parse --short HEAD)"
+          echo "::set-output name=ldflags::-s -w"
           echo "::set-output name=git-short-sha::$(git rev-parse --short HEAD)"
 
   generate-metadata-file:
```

Comment:
Is it bad that this is called from `./consul-lambda/consul-lambda-registrator` since it will be used for the extension too?